app/src/main/java/app/revanced/manager/ui/viewmodel/PatcherViewModel.kt

```diff
@@ -286,11 +369,20 @@ class PatcherViewModel(
                     }
                 }
             }
-        } finally {
+        } catch(_: Exception) {
             isInstalling = false
```

Comment:
The exception should be rethrown or handled in some other way. Will `isInstalling` be unset properly with root installs?

Reply:
OK, it now prints a message just like when the root installation fails. And if the root installation fails it also sets `isInstalling` to false